packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -59,23 +79,17 @@ class BaseFileLink(BaseModel):
     store: Union[str, int] = Field(
         ...,
         description="The store identifier: '0' or 0 for simcore S3, '1' or 1 for datcore",
-        examples=["0", 1],
     )
 
     path: str = Field(
         ...,
         regex=r"^.+$",
         description="The path to the file in the storage provider domain",
-        examples=[
-            "N:package:b05739ef-260c-4038-b47d-0240d04b0599",
-            "94453a6a-c8d4-52b3-a22d-ccbf81f8d636/d4442ca4-23fd-5b6b-ba6d-0b75f711c109/y_1D.txt",
```

Comment:
Why removing this specific example? the datcorlink and simcorelink are very different.

Reply:
I moved them to the concrete subclasses as a complete example ...this way they can also be tested